README.md

```diff
@@ -16,6 +16,31 @@ Hosted runners come with many bundled packages, see
 [Available Environments](https://github.com/actions/virtual-environments#available-environments)
 for details.
 
+### pre-commit
+
+You can execute pre-commit step in a dedicated new workflow:
+
+```yml
+name: pre-commit
+
+on:
+  pull_request:
+    branches: [ master ]
+  push:
```

Comment:
there is a missing branches here

Reply:
is intentional, defaults build all branches (and tags)